arm/Microsoft.ServiceBus/namespaces/ipFilterRules/deploy.bicep

```diff
@@ -0,0 +1,46 @@
+@description('Required. Name of the parent Service Bus Namespace for the Service Bus Queue.')
+@minLength(6)
+@maxLength(50)
+param namespaceName string
+
+@description('Optional. The name of the ip filter rule')
+param name string = filterName
```

Comment:
Something is strange here. Its depending on a required input?

Reply:
Jap. The name is optional. The logic was that you can provide a dedicated name and if not the logic just re-uses the filtername